nf_core/pipeline-template/{{cookiecutter.name_noslash}}/modules/nf-core/multiqc.nf

```diff
@@ -5,35 +5,19 @@ if (!(workflow.runName ==~ /[a-z]+_[a-z]+/)) {
     custom_runName = workflow.runName
 }
 
-// Channel.from(summary.collect{ [it.key, it.value] })
-//     .map { k,v -> "<dt>$k</dt><dd><samp>${v ?: '<span style=\"color:#999999;\">N/A</a>'}</samp></dd>" }
-//     .reduce { a, b -> return [a, b].join("\n            ") }
-//     .map { x -> """
-//     id: '{{ cookiecutter.name_noslash }}-summary'
-//     description: " - this information is collected when the pipeline is started."
-//     section_name: '{{ cookiecutter.name }} Workflow Summary'
-//     section_href: 'https://github.com/{{ cookiecutter.name }}'
-//     plot_type: 'html'
-//     data: |
-//         <dl class=\"dl-horizontal\">
-//             $x
-//         </dl>
-//     """.stripIndent() }
-//     .set { ch_workflow_summary }
-
 /*
  * MultiQC
  */
 process MULTIQC {
     publishDir "${params.outdir}/multiqc", mode: params.publish_dir_mode
 
     input:
-    path (multiqc_config) from ch_multiqc_config
-    path (mqc_custom_config) from ch_multiqc_custom_config.collect().ifEmpty([])
+    path (multiqc_config)
```

Comment:
```suggestion
    path multiqc_config
```

Reply:
Do we need the braces around all of these parameters?

Reply:
works fine without as well!